File: sem2/ADF-I/java-1/Pretest/src/data/Employee.java

package data;


public class Employee extends Person {
    public int salary, workedDays;
    public String position;

    public Employee(String id, String name, int yob,int salary,int workedDays,String position) {
        super(id, name, yob);
        this.salary = salary;
        this.workedDays = workedDays;
        this.position = position;
    }
    
    
    
}

File: sem2/ap-assignments/AP/d03_Inheritance/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace d03_Inheritance
{
    public class Employee : Person
    {
        public string position;
        public int salary;

        public override string ToString()
        {
            var s = gender ? "nam" : "nu";
            return string.Format($"{id},{name},{s},{position},{salary}");
        }
    }
}
